hibernate-core/src/test/java/org/hibernate/test/pagination/PaginationTest.java

```diff
@@ -118,6 +124,42 @@ public void testOffset() {
 		cleanupTestData();
 	}
 
+	/**
+	 * HHH-951: test that setting maxResults does not cause "ORA-00918: column ambiguously defined" on certain queries
+	 * 
+	 * @author Piotr Findeisen <piotr.findeisen@gmail.com>
+	 */
+	@Test
+	@RequiresDialectFeature(
+			value = DialectChecks.SupportLimitCheck.class,
+			comment = "Dialect does not support limit"
+	)
+	public void testLimitWithExpreesionAndFetchJoin() {
+		session = openSession();
```

Comment:
session is private in BaseNonConfigCoreFunctionalTestCase
so it should be Session session = openSession();